.github/workflows/m68k.yml

```diff
@@ -36,13 +36,20 @@ jobs:
         ]
 
     steps:
+    - uses: actions/checkout@v3
+
+    # `rustup show` installs from rust-toolchain.toml
+    - name: Setup rust toolchain
+      run: rustup show
+
+    - name: Setup rust cache
+      uses: Swatinem/rust-cache@v2
+
     - name: Install packages
       run: |
         sudo apt-get update
         sudo apt-get install qemu qemu-user-static
 
-    - uses: actions/checkout@v3
```

Comment:
Was it intentional to remove this?

Reply:
I moved it to the top of the block to be consistent with the other files. Is there a reason it was lower?

Reply:
Ah, sorry, I missed this.
Move it should be fine.